src/core/event.js

```diff
@@ -482,9 +482,19 @@ Candy.Core.Event = (function(self, Strophe, $) {
 			}
 
 			if(directInvite.length > 0) {
+				fromUser = Candy.Core.getRoster().get(msg.attr('from'));
+				room = Candy.Core.getRoom(directInvite.attr('jid'));
```

Comment:
Ah, right. What I've been doing in this case is just keeping the `roomJid` as a string and leaving as is.